coordinator/ethereum/gas-pricing/static-cap/src/test/kotlin/net/consensys/linea/ethereum/gaspricing/staticcap/ExtraDataV1PricerServiceTest.kt

```diff
@@ -49,9 +49,9 @@ class ExtraDataV1PricerServiceTest {
   )
 
   @Test
-  @Timeout(2, timeUnit = TimeUnit.SECONDS)
+  @Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
   fun start_startsPollingProcess(vertx: Vertx, testContext: VertxTestContext) {
-    val pollingInterval = 10.milliseconds
+    val pollingInterval = 50.milliseconds
```

Comment:
I don't think we need to increase the `pollingInterval` here either, the `pollingInterval` in `ExtraDataV1PricerService` is just used as the delay for vertx.setTimer() between each action run